src/components/Blurb.vue

```diff
@@ -1,4 +1,4 @@
-<!-- A Blurb is a span that has styling and possibly a tooltip. -->
+<!-- A Blurb is a span that has styling and a tooltip. -->
```

Comment:
I thought we figured out that every text snippet is contained in a `Blurb`, whether or not it has a flag attached to it. Has that changed?

Reply:
@dylanjwu , can you clarify on this?

Reply:
@ProfJanetDavis  Yes, you're right. I've changed the comment accordingly.